src/crepe/api/Config.h

```diff
@@ -35,6 +35,11 @@ class Config {
 		 */
 		bool color = true;
 	} log;
+
+	//! Save manager
+	struct {
+		std::string location = "save.crepe.db";
```

Comment:
Is deze locatie/ database altijd hard coded of kan deze aangepast worden door de user?

Reply:
De locatie is niet const dus kan aangepast worden tot de database geopend is. Ik zet het erbij in de doxygen documentatie